v2/pkg/cli/mock-delete.go

```diff
@@ -0,0 +1,34 @@
+package cli
+
+import (
+	"context"
+
+	_ "github.com/distribution/distribution/v3/registry/storage/driver/filesystem"
+	"github.com/openshift/oc-mirror/v2/pkg/api/v1alpha3"
+)
+
+type MockDelete struct{}
+
+func (o *MockDelete) ReadDeleteMetaData() ([]v1alpha3.CopyImageSchema, error) {
```

Comment:
Since we are not going to modify the method variable (o *MockDelete), it is not necessary to use a pointer.